### PR TITLE
Adding charts ezd-backend and ezd-crd for ezd-rp from Linux Polska

### PR DESCRIPTION
**Pull request by dariuszbolkowski**
Our package contains set of necessary services which allow to lunch EZD-RP application developed by NASK on Kubernetes using [Helm](https://github.com/helm/helm).

Pull request contains two helm charts:
- ezd-crd
- ezd-backend

### CODE REVIEW

**Comment by recena** [comment]
Since there is a Helm Charts Index Repository (https://linuxpolska.github.io/ezd-rp/index.yaml), you can configure it in Rancher Manager and install the Helm Charts.

📚 Rancher Manager [documentation](https://ranchermanager.docs.rancher.com/v2.0-v2.4/how-to-guides/new-user-guides/helm-charts-in-rancher/adding-catalogs#add-private-githelm-chart-repositories) to do it.

**Comment by nicholasSUSE** [comment]
> Our package contains set of necessary services which allow to lunch EZD-RP application developed by NASK on Kubernetes using [Helm](https://github.com/helm/helm).
> 
> Pull request contains two helm charts:
> 
> * ezd-crd
> * ezd-backend

Hello,  @dariuszbolkowski 

Please provide a screenshot of a working installation of your Chart in Rancher. 

I tried here on my end and could not install it: 
image


Did you follow this procedure? 
https://github.com/rancher/partner-charts/blob/main-source/README.md#testing-your-configuration

---

Also, your chart needs an initial configuration by entering values for Redis, Postgres etc... This is not a PR blocker, it would be good to have this information in your README.md file for your users.

**Comment by dariuszbolkowski** [comment]
Hi @nicholasSUSE ,

Firstly you need to install operators which is "ezd-crd" package. After that you will be able to install "ezd-backend" package.

Best regards,
Dariusz Bółkowski

**Comment by nicholasSUSE** [comment]
> Hi @nicholasSUSE ,
> 
> Firstly you need to install operators which is "ezd-crd" package. After that you will be able to install "ezd-backend" package.
> 
> Best regards, Dariusz Bółkowski

Hello @dariuszbolkowski 
Your PR looks good to me. 
I will be approving. 

Just another suggestion:
Normally the charts come configured with the 1st option defined here: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

There is no problem having to install the CRDs separately first, this is just user experience. 

I have tested your chart on Rancher 2.9 and K8s 1.26 with the `crds dir`. 
It worked.

**Review by nicholasSUSE — APPROVED** [review]

**Review by recena — APPROVED** [review]

**Comment by recena** [comment]
@dariuszbolkowski Could you force an empty commit?

**Comment by nicholasSUSE** [comment]
> @dariuszbolkowski Could you force an empty commit?

Agreed, I have just tested with a dummy Pull Request and the CI ran successfully. 
@dariuszbolkowski 
Please do a force commit.

File: packages/linuxpolska/ezd-backend/upstream.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+HelmRepo: https://linuxpolska.github.io/ezd-rp
+HelmChart: ezd-backend
+Vendor: Linux Polska
+DisplayName: LP Backend for EZD RP 
+ChartMetadata:
+  kubeVersion: '>=1.19-0'
+  icon: https://linuxpolska.com/logo/LinuxPolska-icon.png
```
[the remark this answers is not in the document]
Comment (recena) [reply packages/linuxpolska/ezd-backend/upstream.yaml:7]: In both helm charts.

[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply packages/linuxpolska/ezd-backend/upstream.yaml:7]: @recena  Will do this shortly. Thanks.

[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply packages/linuxpolska/ezd-backend/upstream.yaml:7]: It is done now. Pleasy veryfi. 
Thank you.

**Review by recena — APPROVED** [review]

File: packages/linuxpolska/ezd-backend/upstream.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (recena) [reply packages/linuxpolska/ezd-backend/upstream.yaml:7]: There is a [conflict](https://github.com/rancher/partner-charts/actions/runs/7985109955/job/21804104068?pr=963) with the rebase. Could you do it manually?

[the remark this answers is not in the document]
Comment (dariuszbolkowski) [reply packages/linuxpolska/ezd-backend/upstream.yaml:7]: I have done rebase. I hope it will works.

**Pull request merged**